examples/revertrisk_examples.py

```diff
@@ -0,0 +1,31 @@
+import json
+import requests
+
+def revert_risk_api_request(language: str, revision_id: int, use_auth: bool, access_token: str, user_agent: str):
```

Comment:
These could be optional `use_auth: bool, access_token: str` with a default value for `use_auth=False`